scrollUp.jsx

```diff
@@ -69,6 +69,7 @@ var ScrollUp = React.createClass({
     },
 
     handleScroll: function () {
+      console.log('got here')
```

Comment:
This is only for imagine how it works, right? So can you remove it to accept your pull request.